src/components/sbb-navigation-marker/sbb-navigation-marker.tsx

```diff
@@ -55,73 +49,81 @@ export class SbbNavigationMarker implements ComponentInterface {
     this._setMarkerPosition();
   }
 
-  public connectedCallback(): void {
-    this._navigationMarkerResizeObserver.observe(this._element);
+  public override connectedCallback(): void {
+    super.connectedCallback();
+    this._navigationMarkerResizeObserver.observe(this);
     this._readActions();
   }
 
-  public disconnectedCallback(): void {
+  public override disconnectedCallback(): void {
+    super.disconnectedCallback();
     this._navigationMarkerResizeObserver.disconnect();
   }
 
-  @Method()
-  public async select(action: HTMLSbbNavigationActionElement): Promise<void> {
-    await this.reset();
+  public select(action: SbbNavigationAction): void {
+    this.reset();
     action.active = true;
     this._currentActiveAction = action;
     this._hasActiveAction = true;
-    this._setMarkerPosition();
+    setTimeout(() => this._setMarkerPosition());
```

Comment:
😢

Reply:
I know, unfortunately here if I do not use the timeout the query `this.shadowRoot.querySelector('[data-active]')` fails and does not work...